Backpack/Skeleton/Classes/BPKSkeleton.swift

```diff
@@ -0,0 +1,166 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import UIKit
+import SwiftUI
+
```

Comment:
```suggestion
enum BPKSkeletonConstants {
    static let backgroundColor = BPKColor.dynamicColor(
        withLightVariant: BPKColor.skyGrayTint06,
        darkVariant: BPKColor.blackTint02
    )
    
    static let defaultColor = BPKColor.dynamicColor(
        withLightVariant: BPKColor.white,
        darkVariant: BPKColor.black
    )
}
```

And then use BPKSKeletonConstants.defaultColor where we would use BPKSkeleton.defaultColor / same for the bgColor.